packages/wrangler/src/generate/index.ts

```diff
@@ -38,81 +38,31 @@ type GenerateArgs = StrictYargsOptionsToInterface<typeof generateOptions>;
 
 // Originally, generate was a rust function: https://github.com/cloudflare/wrangler-legacy/blob/master/src/cli/mod.rs#L106-L123
 export async function generateHandler(args: GenerateArgs) {
-	// somehow, `init` marks name as required but then also runs fine
-	// with the name omitted, and then substitutes it at runtime with ""
-	// delegate to `wrangler init` if no template is specified
```

Comment:
Need to rip out `initHandler` too and just make it simply proxy over to C3. Bunch of things to rip out there too.